docker/iml-action-runner.dockerfile

```diff
@@ -4,7 +4,8 @@ FROM ubuntu
 COPY --from=builder /build/target/release/iml-action-runner /usr/local/bin
 
 RUN apt-get update \
-  && apt-get install -y curl
+  && apt-get install -y curl \
```

Comment:
There is some commonality between some of these images. I'll take care of moving the common pieces to a base image.